stable/concourse/values.yaml

```diff
@@ -678,6 +678,16 @@ concourse:
       ##
       useCaCert: false
 
+    sidecarContainers:
+    - name: atc-syslog
+      image: splunk/scs
+      ## See https://splunk-connect-for-syslog.readthedocs.io/en/master/gettingstarted/docker-systemd-general/#configure-the-sc4s-environment
+      env:
+      - name: SPLUNK_HEC_URL
+        value: https://http-inputs-opendoor.splunkcloud.com:443
+      - name: SPLUNK_HEC_TOKEN
+        value: od_secret:concourse/engineering/concourse-web/splunk_hec_token#value
```

Comment:
The vault path concourse web has access to is `od_secret:secret/kubernetes/concourse/concourse-web/`

The `value` key is a concourse specific naming convention.  This can be changed to something like `od_secret:secret/kubernetes/concourse/concourse-web/splunk#hec_token`.

`vault write secret/kubernetes/concourse/concourse-web/splunk hec_token=<token-value>`